Sources/JavaKit/JavaKitVM/JavaVirtualMachine.swift

```diff
@@ -268,4 +280,8 @@ extension JavaVirtualMachine {
       }
     }
   }
+
+  enum JavaKitError: Error {
+    case classPathEntryNotFound(entry: String, classPath: String)
```

Comment:
```suggestion
    case classPathEntryNotFound(entry: String, classPath: [String])
```

Reply:
Fixing compilation

Reply:
Oh, it was supposed to be `colonSeparatedClassPath` – must have dropped that diff somehow. But I suppose the colon separation is an implementation detail. I'll do as you suggest above.